packages/html-reporter/src/testCaseView.css

```diff
@@ -73,4 +73,14 @@
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
+}
+
+.annotation-copy-button{
+  padding-left: 3px;
+  display: none;
+}
+
+.test-case-annotation:hover .annotation-copy-button{
```

Comment:
Sounds like you would need this dance for every copy button application. Would it be possible to do something like this to encapsulate these styles:

```
<CopyToClipboardContainer description={description}>
  <span className='annotation-copy-button'/>
</CopyToClipboardContainer>
```
```

Reply:
Not every copy button , just for this annotation copy button .  We want to display the copy button only when we hover over the text . Could you please help me with how your approach would help in this use case ?